From cd6e5f49b99a587b256857b66d37202ae5cfe027 Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Wed, 11 Jan 2012 01:51:52 +0100
Subject: [PATCH] shutdown: add link to root storage daemon text (cherry
 picked from commit
 bd1a69818042e85e24ec3adaf5eb3ac30ab1d9fd)

---
 src/shutdown.c |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/src/shutdown.c b/src/shutdown.c
index 46b5aea..d157e0f 100644
--- a/src/shutdown.c
+++ b/src/shutdown.c
@@ -81,7 +81,9 @@ static bool ignore_proc(pid_t pid) {
                 return true;
 
         /* Processes with argv[0][0] = '@' we ignore from the killing
-         * spree. */
+         * spree.
+         *
+         * http://www.freedesktop.org/wiki/Software/systemd/RootStorageDaemons */
         if (count == 1 && c == '@')
                 return true;
 
